### PR TITLE
Adds lobby to the gameInfo page

### LINKED ISSUE #621: Public rooms loading causes a flash on page load for /

### PR DESCRIPTION
**Pull request by vdfdev**
Implements first features as discussed on #941.  Fixes #621.

Related improvements include:
- New room button is now in the shape of a card in the end of the carrousel
- Move lobby to the bottom of the home page 
- Add height limit to the instruction (so users can see the video above the fold)
- Several alignments improvements
- Removes padding inside carrousel

Before:

image

After:

image

Before:

image

After:

image

#### Checklist

* [x] Use a separate branch in your local repo (not `master`).
* [x] I've read and agree with the [contribution guidelines](https://www.freeboardgames.org/docs/?path=/story/documentation-contributing--page).

### CODE REVIEW

**Comment by richardaum** [comment]
LGTM overall, but it is hard to correlate the changes that you mention on description with the code. I tried to guide myself using the commits, but they are also unrelated.

**Review by richardaum — APPROVED** [review]

File: web/src/infra/lobby/LobbyCarousel.tsx [hunk 1]
```diff
@@ -44,67 +48,69 @@ class LobbyCarousel extends React.Component<Props, State> {
...
-      return (
-        <Carousel>
-          <CircularProgress className={css.carouselCenter} />
-        </Carousel>
-      );
+      return <CircularProgress className={css.carouselCenter} />;
     } else if (this.state.error) {
       return (
-        <Typography component="h2" variant="body2" className={css.message}>
+        <Typography component="h2" variant="body2" className={css.messag}>
```
Comment (richardaum) [inline web/src/infra/lobby/LobbyCarousel.tsx:93]: Why not `message`?

File: web/src/infra/lobby/NewRoomCard.module.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.CardWrapper {
+    width: 219px;
+    min-width: 219px;
```
Comment (richardaum) [inline web/src/infra/lobby/NewRoomCard.module.css:3]: Probably we don't need a min-width considering we already have a width.

File: web/src/infra/lobby/LobbyCarousel.tsx (hunk 1 above)
Comment (vdfdev) [reply web/src/infra/lobby/LobbyCarousel.tsx]: That was a typo, good catch!

File: web/src/infra/lobby/NewRoomCard.module.css (hunk 2 above)
Comment (vdfdev) [reply web/src/infra/lobby/NewRoomCard.module.css:3]: We do, I thought the same but somehow the card collapse on mobile if we dont have mind width.

**Comment by vdfdev** [comment]
@richardaum thanks for the review, next time I will try to be more descriptive on the commits. Still getting used to git as I use something else at work :P

**Pull request merged**